nifi-registry/nifi-registry-core/nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/security/NiFiRegistrySecurityConfig.java

```diff
@@ -166,8 +166,7 @@ private AuthenticationEntryPoint http401AuthenticationEntryPoint() {
 
             // return a 401 response
             final int status = HttpServletResponse.SC_UNAUTHORIZED;
-            logger.info("Client could not be authenticated due to: {} Returning 401 response.", authenticationException.toString());
```

Comment:
Recommend reverting this change, but adding a message to the `debug` call along the lines of `debug("HTTP 401 Unauthorized", authenticationException)`